### PR TITLE
"Refreshing server adapter list" can be improved to not wait as long

### PR DESCRIPTION
**Pull request by rawagner**

### CODE REVIEW

File: plugins/org.eclipse.reddeer.eclipse/src/org/eclipse/reddeer/eclipse/wst/server/ui/wizard/NewServerWizard.java [hunk 1]
```diff
@@ -35,15 +37,28 @@ public NewServerWizard() {
...
 
 	@Override
 	public void cancel() {
 		// workaround due to JBDS-3596
-		new WaitUntil(new JobIsKilled("Refreshing server adapter list"), TimePeriod.LONG, false);
+		handleRefereshingServersJob();
 		super.cancel();
 	}
 	
+	private void handleRefereshingServersJob(){
```
Comment (apodhrad) [inline plugins/org.eclipse.reddeer.eclipse/src/org/eclipse/reddeer/eclipse/wst/server/ui/wizard/NewServerWizard.java]: Please avoid duplicates of the same logic. Consider creating a new WaitCondition, e.g. ServerAdaptersAreRefreshing.

Comment (apodhrad) [inline plugins/org.eclipse.reddeer.eclipse/src/org/eclipse/reddeer/eclipse/wst/server/ui/wizard/NewServerWizard.java]: @rawagner ^^^

**Comment by apodhrad** [comment]
testPR

**Comment by mlabuda** [comment]
Please fix conflicts.

**Comment by apodhrad** [comment]
@rawagner what's the state of this PR?

**Comment by apodhrad** [comment]
@rawagner what's the state of this PR?

**Pull request closed without merging**